tests/shared/src/test/scala/scala/meta/tests/parsers/TopLevelTermSuite.scala

```diff
@@ -0,0 +1,82 @@
+package scala.meta.tests.parsers
+
+import scala.compat.Platform.EOL
+import scala.meta._
+
+import munit._
+
+class TopLevelTermSuite extends FunSuite {
+  test("allowToplevelTerms simple") {
+    implicit val dialect: Dialect = dialects.Scala3.withAllowToplevelTerms(true)
+
+    val sourceString = """def foo(x: Int): Int = x
+                          |foo(x)
+""".trim.stripMargin
+
+    val tree = sourceString.parse[Source].get
+
+    assert(tree.syntax == sourceString)
+    assert(
+      tree.structure ==
+        """Source(List(
+          |Defn.Def(Nil, Term.Name("foo"), Nil, List(List(Term.Param(Nil, Term.Name("x"), Some(Type.Name("Int")), None))), Some(Type.Name("Int")), Term.Name("x")), 
+          |Term.Apply(Term.Name("foo"), List(Term.Name("x")))))
+          |""".trim.stripMargin.split('\n').mkString("")
```

Comment:
is it the same as `replace("\n", "")`?

Reply:
Probably. I stole this from another [suite](https://github.com/scalameta/scalameta/blob/main/tests/shared/src/test/scala/scala/meta/tests/parsers/SbtSuite.scala#L83). Happy to replace.